apps/code/catalog.de.i18n

```diff
@@ -12,186 +12,186 @@ PythonSingleQuote = "Einfaches Anführungszeichen"
+PythonLog10 = "Logarithmus Basis 10"
+PythonLog2 = "Logarithmus Basis 2"
+PythonMathFunction = "Präfix Mathematikmodulfunktion"
+PythonMatplotlibPyplotFunction = "matplotlib.pyplot Modulpräfix"
+PythonMax = "Höchstwert"
+PythonMin = "Mindestwert"
+PythonModf = "Bruch- und Ganzzahlteile von x"
+PythonMonotonic = "Wert einer monotonen Uhr"
+PythonOct = "Ganzzahl in Oktal konvertieren"
+PythonPhase = "Phase von z"
+PythonPlot = "Linien y gegen x zeichnen"
+PythonPolar = "z in Polarkoordinaten"
+PythonPop = "Das letzte Element entfernen und ausgeben"
+PythonPower = "x erhöht auf die Potenz y"
+PythonPrint = "Objekt drucken"
+PythonRadians = "Konvertiere x von Grad in Bogenmaß"
+PythonRandint = "Zufällige Ganzzahl in [a,b]"
+PythonRandom = "Gleitkommazahl in [0,1]"
+PythonRandomFunction = "Zufälliges Modulfunktionspräfix"
+PythonRandrange = "Zufallszahl im Bereich (Start, Stopp)"
+PythonRangeStartStop = "Liste von Anfang bis Ende-1"
+PythonRangeStop = "Liste von 0 bis Stopp-1"
+PythonRect = "In kartesische Koordinaten konvertieren"
+PythonRemove = "Entfernen des ersten Vorkommens von x"
+PythonReverse = "Elemente der Liste umkehren"
+PythonRound = "Auf n Ziffern runden"
+PythonScatter = "Streudiagramm von y gegen x zeichnen"
+PythonSeed = "Zufallszahlengenerator initialisieren"
+PythonSetPixel = "Farbpixel (x,y)"
+PythonShow = "Figur zeigen"
+PythonSin = "Sinus"
+PythonSinh = "Hyperbolischer Sinus"
+PythonSleep = "Ausführung für t Sekunden unterbrechen"
+PythonSort = "Liste sortieren"
+PythonSqrt = "Quadratwurzel"
+PythonSum = "Elemente einer Liste summieren"
+PythonTan = "Tangente"
+PythonTanh = "Hyperbolische Tangente"
+PythonText = "Text an (x,y) Koordinaten anzeigen"
+PythonTimeFunction = "Zeitmodul-Funktionspräfix"
+PythonTrunc = "x auf eine Ganzzahl gekürzt"
+PythonTurtleBackward = "Um x Pixel rückwärts bewegen"
+PythonTurtleCircle = "Kreis mit Radius r Pixel"
+PythonTurtleColor = "Stiftfarbe einstellen"
+PythonTurtleColorMode = "Stellen Sie den Farbmodus auf 1.0 oder 255 ein"
+PythonTurtleForward = "Vorwärts um x Pixel bewegen"
+PythonTurtleFunction = "Funktionspräfix des Schildkröten-Moduls"
+PythonTurtleGoto = "Zu (x,y) Koordinaten verschieben"
+PythonTurtleHeading = "Aktuelle Überschrift zurückgeben"
+PythonTurtleHideturtle = "Verstecke die Schildkröte"
+PythonTurtleIsdown = "Gibt true (wahr) zurück, wenn der Stift angelegt ist"
+PythonTurtleLeft = "Um ein Grad nach links abbiegen"
+PythonTurtlePendown = "Ziehen Sie den Stift nach unten"
+PythonTurtlePensize = "Linienstärke auf x Pixel setzen"
+PythonTurtlePenup = "Stift anheben"
+PythonTurtlePosition = "Aktuellen (x,y) Speicherort zurückgeben"
+PythonTurtleReset = "Zeichnung zurücksetzen"
+PythonTurtleRight = "Um ein Grad nach rechts abbiegen"
+PythonTurtleSetheading = "Ausrichtung auf einen Grad setzen"
+PythonTurtleSetposition = "Schildkröte positionieren"
+PythonTurtleShowturtle = "Schildkröte zeigen"
+PythonTurtleSpeed ​​= "Zeichengeschwindigkeit zwischen 0 und 10"
```

Comment:
```suggestion
PythonTurtleSpeed = "Zeichengeschwindigkeit zwischen 0 und 10"
```
The first space after `PythonTurtleSpeed` was wrong. Look at this output when copying and escaping the chars in python :
```
>>> "PythonTurtleSpeed ​​= \"Zeichengeschwindigkeit zwischen 0 und 10\""
'PythonTurtleSpeed \u200b\u200b= "Zeichengeschwindigkeit zwischen 0 und 10"'
```